examples/led_fade/config/config.exs

```diff
@@ -0,0 +1,9 @@
+use Mix.Config
+
+i2c_implementation =
+  case Mix.env() do
+    :prod -> ElixirALE.I2C
+    _ -> GrovePi.I2C
+  end
+
+config :grovepi, i2c: i2c_implementation
```

Comment:
I would remove that. By default we fall back to ElixirALE if the i2c is not configured. It might just add confusion.